processor/spanmetricsprocessor/processor.go

```diff
@@ -221,21 +233,42 @@ func (p *processorImp) ConsumeTraces(ctx context.Context, traces pdata.Traces) e
 // writes the raw metrics data into the metrics object.
 func (p *processorImp) buildMetrics() *pdata.Metrics {
 	m := pdata.NewMetrics()
-	ilm := m.ResourceMetrics().AppendEmpty().InstrumentationLibraryMetrics().AppendEmpty()
-	ilm.InstrumentationLibrary().SetName("spanmetricsprocessor")
 
-	p.lock.RLock()
-	p.collectCallMetrics(ilm)
-	p.collectLatencyMetrics(ilm)
-	p.lock.RUnlock()
+	rms := m.ResourceMetrics()
+	for key := range p.resourceAttrList {
+		p.lock.RLock()
+		raKey := metricKey(key)
+		resourceAttributesMap := p.metricKeyToDimensions[raKey]
```

Comment:
nit, the naming is a bit inconsistent here. Maybe call it `raMap`?

Reply:
I renamed all the keys to resourceAttrKey since I think it's easier to read. LMK if you disagree.